src/pages/membership/components/editYouthProfile/EditYouthProfile.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import { useMutation, useQuery } from '@apollo/react-hooks';
+import { loader } from 'graphql.macro';
+import { useHistory } from 'react-router';
+
+import {
+  AddressType,
+  Language,
+  MembershipDetails as MembershipDetailsData,
+  PhoneType,
+  UpdateMyProfile as UpdateMyProfileData,
+  UpdateMyProfileVariables,
+  YouthLanguage,
+} from '../../../../graphql/generatedTypes';
+import YouthProfileForm, {
+  FormValues,
+} from '../youthProfileForm/YouthProfileForm';
+import styles from './EditYouthProfile.module.css';
+import NotificationComponent from '../../../../common/notification/NotificationComponent';
+
+const MEMBERSHIP_DETAILS = loader('../../graphql/MembershipDetails.graphql');
+const UPDATE_PROFILE = loader('../../graphql/UpdateMyProfile.graphql');
+
+type Props = {};
+
+function EditYouthProfile(props: Props) {
+  const [showNotification, setShowNotification] = useState<boolean>(false);
+  const { data, loading: loadingProfile } = useQuery<MembershipDetailsData>(
+    MEMBERSHIP_DETAILS
+  );
+
+  const [updateProfile, { loading: saveLoading }] = useMutation<
+    UpdateMyProfileData,
+    UpdateMyProfileVariables
+  >(UPDATE_PROFILE);
+
+  const history = useHistory();
+  const youthProfile = data?.youthProfile;
+
+  const handleOnValues = (formValues: FormValues) => {
+    const variables: UpdateMyProfileVariables = {
+      input: {
+        profile: {
+          firstName: formValues.firstName,
+          lastName: formValues.lastName,
+          language: formValues.profileLanguage,
+          updateAddresses: [
+            {
+              address: formValues.address,
+              postalCode: formValues.postalCode,
+              city: formValues.city,
+              addressType: AddressType.OTHER,
+              primary: true,
+              id: youthProfile?.profile?.primaryAddress?.id,
+            },
+          ],
+          updatePhones: [
+            {
+              phone: formValues.phone,
+              phoneType: PhoneType.OTHER,
+              primary: true,
+              id: youthProfile?.profile?.primaryPhone?.id,
+            },
+          ],
+          youthProfile: {
+            birthDate: youthProfile?.birthDate,
+            schoolName: formValues.schoolName,
+            schoolClass: formValues.schoolClass,
+            approverFirstName: formValues.approverFirstName,
+            approverLastName: formValues.approverLastName,
+            approverPhone: formValues.approverPhone,
+            approverEmail: formValues.approverEmail,
+            languageAtHome: formValues.languageAtHome,
+          },
+        },
+      },
+    };
+
+    updateProfile({ variables })
+      .then(() => {
+        history.push('/membership-details');
+      })
+      .catch((error: Error) => setShowNotification(true));
```

Comment:
Does setShowNotification push errors to sentry? If not, did you consider pushing the errors to sentry - it might help us catch errors caused by bugs in the API for example.

Reply:
Oh damn, that was my intention but seems that I forgot to do it...